src/main/java/com/google/maps/DirectionsClient.java

```diff
@@ -1,10 +1,23 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package com.google.maps;
 
 import java.util.List;
 
-/**
- * Contract for sending GET requests to the Google Directions API.
- */
+/** Contract for sending GET requests to the Google Directions API. */
 public interface DirectionsClient {
-  List<String> getDirections(String origin, String destination, String[] waypoints);
+  List<String> getDirections(String origin, String destination, String[] waypoints)
+      throws Exception;
```

Comment:
You should specify what type of exception you are throwing - Here you would likely have to specify all three possible types (`ApiException, InterruptedException, IOException`)

Reply:
Fixed, thanks!

Reply:
`RESOLVED`